src/Microsoft.Azure.WebJobs.Host/Extensions/MethodMetadata .cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    /// <summary>
+    /// Represents method metadata.
+    /// </summary>
+    public class MethodMetadata
+    {
+        /// <summary>
+        /// Gets or sets whether this method is disabled.
```

Comment:
>sets [](start = 20, length = 4)

This isn't true, right? Setting this to True does not disable the method.  We return a poco, but we ignore any mutations.  
The current behavior is fine, but we should update the comment.